StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -309,6 +312,14 @@ void Renderer::Execute()
 	renderTask->SetCommandInterfaceIndex(commandInterfaceIndex);
 	m_pThreadPool->AddTask(renderTask, FLAG_THREAD::RENDER);
 
+	// Skybox
+	if (DRAWSKYBOX == true)
+	{
+		m_RenderTasks[RENDER_TASK_TYPE::SKYBOX]->SetBackBufferIndex(backBufferIndex);
+		m_RenderTasks[RENDER_TASK_TYPE::SKYBOX]->SetCommandInterfaceIndex(commandInterfaceIndex);
+		m_pThreadPool->AddTask(m_RenderTasks[RENDER_TASK_TYPE::SKYBOX], FLAG_THREAD::RENDER);
+	}
+
```

Comment:
Add a default skybox instead of this. So that if the game developers haven't created a skyBoxComponent with their desired skyboxTexture, the engine will use a default skybox.